app/controllers/tenants_controller.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 class TenantsController < ApplicationController
-  wrap_parameters Tenant
```

Comment:
why remove this?

Reply:
It's default on rails 7 for all controllers, so not needed anymore. https://github.com/rails/rails/blob/v7.0.5/railties/lib/rails/application/configuration.rb#L259